organize/forms.py

```diff
@@ -0,0 +1,61 @@
+from django import forms
+from django_countries import countries
+from django_countries.fields import LazyTypedChoiceField
+from django_date_extensions.fields import ApproximateDateFormField
+
+
+from core.models import Event
+from core.validators import validate_approximatedate
+from .models import INVOLVEMENT_CHOICES
+
+PREVIOUS_ORGANIZER_CHOICES = (
+    (True, "Yes, I organized Django Girls"),
+    (False, "No, it’s my first time organizing Django Girls"))
+
+
+class PreviousEventForm(forms.Form):
+    has_organized_before = forms.ChoiceField(
+        widget=forms.RadioSelect,
+        choices=PREVIOUS_ORGANIZER_CHOICES)
+    previous_event = forms.ModelChoiceField(
+        queryset=Event.objects.past(), empty_label="Choose event",
```

Comment:
@aniav can we make it so that event on the list displays month and year as well? so we can have: `Django Girls Warsaw #2, May 2017`. Otherwise I bet organizers are gonna be confused. Also, sort it alphabetically?